src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -145,9 +145,16 @@ try
     {
         // TODO: "relative" bold?
         weight = DWRITE_FONT_WEIGHT_BOLD;
+        // Since we are setting the font weight according to the text attribute,
+        // make sure to tell the text format to ignore the user set font weight
+        _fontRenderData->UseUserWeight(false);
+    }
+    else
+    {
+        _fontRenderData->UseUserWeight(true);
     }
 
-    if (drawingContext->useItalicFont)
+    if (drawingContext->useItalicFont || _fontRenderData->DidUserSetItalic())

```

Comment:
sorry, what's this check do? does this promote the `ital` axis back into `STYLE_ITALIC`?

Reply:
This is so `FontFaceWithAttribute` gets called with `DWRITE_FONT_STYLE_ITALIC` if the `"ital"` axis is set, otherwise this happens:

<img width="802" alt="ital" src="https://user-images.githubusercontent.com/26824113/130521003-7dd5cbfa-f502-445c-a9e5-854a4972b999.png">